src/main/resources/api/schemas/calendar.yaml

```diff
@@ -0,0 +1,71 @@
+description: A single calendar
+type: object
+properties:
+  id:
+    description: A unique UUID identifying this calendar
+    type: string
+    format: uuid
+  name:
+    description: A user-provided name used to label this calendar
+    type: string
+  startDate:
+    type: string
+    format: date
+    description: The first effective date (inclusive) of this calendar
```

Comment:
The generated documentation does not describe what the date format means for clients (I don't know if this is a limitation of OpenAPI).

Maybe it could be worth putting the format in the description to avoid ambiguity.

Reply:
(It seems the OpenAPI specification defines these formats, however they are not included in the generated documentation. This seems a somewhat odd choice to me, requiring readers to know about the underlying documentation specification)

Reply:
Agreed; I'll add a note